gui/default/index.html

```diff
@@ -958,7 +958,6 @@ <h4 class="panel-title">
   <script type="text/javascript" src="syncthing/core/notificationDirective.js"></script>
   <script type="text/javascript" src="syncthing/core/pathIsSubDirDirective.js"></script>
   <script type="text/javascript" src="syncthing/core/popoverDirective.js"></script>
-  <script type="text/javascript" src="syncthing/core/selectOnClickDirective.js"></script>
```

Comment:
I've actually found it very annoying, e.g. when I was reporting something on the forum and wanted to copy only a part of the ID, having to fight with the automatic selection, eventually ending up temporarily disabling JS or using developer tools just to be able to copy the text.

This script isn't used in any other places, is it? With a copy to clipboard button present, is there any other real use case for it?

Reply:
I'm working on a Linux desktop OS.  There is not only one clipboard, but whatever text is selected gets copied automatically to the so-called "Primary Selection" buffer.  I can instantly paste it in any other text entry field by a single middle-button click.  Two clicks overall, very efficient, no searching for a paste button or switching to the keyboard.  Same goes for the drag-and-drop usage pattern.

Of course the same can be achieved by a left button triple-click usually to select the whole paragraph / text span.  I don't think the current behavior is a must-have, just that there is no real reason to *not* keep it.



> wanted to copy only a part of the ID, having to fight with the automatic selection, eventually ending up temporarily disabling JS or using developer tools just to be able to copy the text.

That sounds awkward indeed.  Why not cut it down to what you need after pasting?

Reply:
> That sounds awkward indeed. Why not cut it down to what you need after pasting?

Yeah, it'd probably be easier that way. I guess it's just the fact of being prevented from selecting parts of the text that makes it feel wrong. I'd say it has a similar feeling to those sites that use JS to block any text selection. I even have a browser extension that pauses JS on one click specifically to override such behaviour.

Another reason against it would be that the current script isn't compatible with IE11, preventing any text selection there, although this won't be that important once the copy to clipboard button gets implemented (because the button does work in the browser).

Reply:
Marking resolved, because I've decided not to touch that script in this PR. The PR itself is still in the works, so please wait with reviewing until I mark it "ready for review" 🙂.